src/api/api.ts

```diff
@@ -1 +1,7 @@
-export const BASE_URL = 'https://mate.academy/students-api';
+export const BASE_URL = 'https://mate.academy/students-api/';
```

Comment:
```suggestion
export const BASE_URL = 'https://mate.academy/students-api';
```
1. Remove slash, because will be duplicated when using `BASE_URL` in line 4